examples/substrate-to-evm-react-example/README.md

```diff
@@ -0,0 +1,80 @@
+# Substrate to EVM example
+
+This is an example react application to test the transfer of assets from Substrate to EVM. It was built primarily used with [sygma-relayer's local setup](https://github.com/sygmaprotocol/sygma-relayer/)
+
+## Prerequisites
+
+- [Yarn](https://yarnpkg.com/) (version 3.4.1 or higher)
+- [Polkadot{.js} browser extension](https://polkadot.js.org/extension/)
+- [Docker](https://www.docker.com/)
+
+## Getting Started
+
+### 0. Prepare local setup
```

Comment:
There is a quicker way to start example:

```
git clone git@github.com:sygmaprotocol/sygma-relayer.git
cd sygma-relayer

make example
```

Reply:
That is a shortcut for `docker-compose --file=./example/docker-compose.yml up --build`, so it will rebuild the images every time. 
I've updated readmes with this method as quick way, and old one as a "manual"